operator/cmd/syncclusterconfig/sync.go

```diff
@@ -88,6 +97,7 @@ If present and not empty, the $%s environment variable will be set as the cluste
 		},
 	}
 
+	cmd.Flags().StringVar(&usersTxtPath, "users-txt", "/etc/secrets/users/users.txt", "Path to users.txt")
```

Comment:
To be more inline with the config-watcher, should we accept [a glob or directory](https://github.com/redpanda-data/helm-charts/blob/56e8cc7a5e8a34ed1a27c8073490e76493ecda86/charts/redpanda/secrets.go#L303) instead of a single file?